lms/djangoapps/certificates/management/commands/rescind_cert.py

```diff
@@ -0,0 +1,104 @@
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    '''Update the status of a single user certificate for a given user in a given course'''
+    help = u'Rescind a certificate for a particular user in a particular course.'
+
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '-c', '--course',
+            metavar='COURSE_ID',
+            dest='course',
+            default=False,
+            help='The course id (e.g., mit/6-002x/circuits-and-electronics) of the course in which the '
+                 'certificate of the student should be updated',
+        ),
+        make_option(
+            '-u', '--user',
+            metavar='USERNAME',
+            dest='username',
+            default=False,
+            help='The username or email address for whom grading and certificate status should be updated',
+        ),
+        make_option(
+            '-s', '--status',
+            metavar='STATUS',
+            dest='status',
+            default=CertificateStatuses.unavailable,
+            help='The status to set to the corresponding certificate',
+        ),
+    )
+
+    def handle(self, *args, **options):
+
+        if options['course']:
+            # try to parse out the course from the serialized form
+            try:
+                course_id = CourseKey.from_string(options['course'])
+            except InvalidKeyError:
+                course_id = SlashSeparatedCourseKey.from_deprecated_string(options['course'])
+        else:
+            raise CommandError(u'You must specify a course')
+
+        user = options['username']
+        if not user:
+            raise CommandError(u'The student\'s username is required')
+
+        status = options['status']
+        valid_statuses = [
+            CertificateStatuses.unavailable,
+            CertificateStatuses.downloadable,
+            CertificateStatuses.notpassing,
+        ]
+        if status not in valid_statuses:
+            raise CommandError(
+                u'INVALID STATUS. '
+                'Supported Statuses: `unavailable`, `downloadable`, `notpassing`'
+            )
+
+        if '@' in user:
+            student = User.objects.get(email=user, courseenrollment__course_id=course_id)
+        else:
+            student = User.objects.get(username=user, courseenrollment__course_id=course_id)
+        try:
+            certificate = GeneratedCertificate.objects.get(user=student, course_id=course_id)
+            certificate.status = status
+            certificate.save()
+            cert_saved_msg = (
+                u"""The certificate status for student {student_id}
+                in course '{course_id}'
+                has been set to '{status}'""".format(
+                    student_id=student.id,
+                    course_id=course_id,
+                    status=certificate.status,
+                )
+            )
+            LOGGER.info(cert_saved_msg)
+            print cert_saved_msg
```

Comment:
That said, the print function is preferred to the print statement, the latter having been deprecated [1].

```python
# In Python 2, this import is technically optional, though it is preferred.
# In Python 3, the print statement is no longer supported.
from __future__ import print_function

print(cert_saved_msg)
```

[1] https://docs.python.org/3/whatsnew/3.0.html#print-is-a-function